routes/apiRoutes.js

```diff
@@ -174,18 +138,72 @@ module.exports = function (app) {
 
   // Initialize the function on app start up
   app.on('listening', function () {
-    var dailyTask = schedule.scheduleJob('0 0 * * *', function () {
-      console.log("======================= DAILY TASK RUNNER running at: " + moment().format() + " ======================");
-      db.UserProfile.findAll({}).then((users) => {
-        // console.log(JSON.stringify(users));
-        users.map((user) => { console.log(user.username); });
-      })
-      .catch(console.log);
-    });
+    dailyTask();
 });  
 
   function checkWeatherInterval() {
     console.log("Checking the weather now....." + moment().format());
   }
   setInterval(checkWeatherInterval, 5000);
 };
+
+var dailyTask = schedule.scheduleJob('0 0 * * *', function () {
+  console.log("======================= DAILY TASK RUNNER running at: " + moment().format() + " ======================");
+  db.UserProfile.findAll({}).then((users) => {
+    users.map((user) => {
+      var today = moment().format('dddd');
+      var HHmmArray = user.timePreference.today.split(":");
+      var scheduleDayTime = HHmmArray[1] + " " + HHmmArray[0] + " * * *";
+      schedule.scheduleJob(scheduleDayTime, function (phoneNumber, zipcode) {
```

Comment:
@Exxodus-sama I think you have a double function in here which may be why you needed to bind today in there.  I probably confused you by giving multiple examples in the prototype branch.  I'll approve this then we can clean it up.